translation.rst

```diff
@@ -297,22 +294,6 @@ To manage these situations, Symfony follows the `ICU MessageFormat`_ syntax by
 using PHP's :phpclass:`MessageFormatter` class. Read more about this in
 :doc:`/translation/message_format`.
 
-.. tip::
```

Comment:
Not sure why we would want to add a tip about something we don't recommend anymore.